app/experimenter/experiments/tests/factories.py

```diff
@@ -216,6 +220,69 @@ def projects(self, create, extracted, **kwargs):
         self.projects.add(*extracted)
 
 
+class ExperimentRapidFactory(ExperimentConstants, factory.django.DjangoModelFactory):
+    type = Experiment.TYPE_RAPID
+    rapid_type = factory.LazyAttribute(
+        lambda o: random.choice(Experiment.RAPID_TYPE_CHOICES)[0]
+    )
+    owner = factory.SubFactory(UserFactory)
+    name = factory.LazyAttribute(lambda o: faker.catch_phrase())
+    slug = factory.LazyAttribute(lambda o: "{}_".format(slugify(o.name)))
+    objectives = factory.LazyAttribute(lambda o: faker.text(1000))
+    audience = factory.LazyAttribute(
+        lambda o: random.choice(list(NIMBUS_DATA["Audiences"].keys()))
+    )
+    features = factory.LazyAttribute(lambda o: list(NIMBUS_DATA["features"].keys()))
+    firefox_min_version = factory.LazyAttribute(
+        lambda o: random.choice(Experiment.VERSION_CHOICES[1:])[0]
+    )
+    firefox_channel = factory.LazyAttribute(
+        lambda o: random.choice(Experiment.CHANNEL_CHOICES[1:])[0]
+    )
+
+    class Meta:
+        model = Experiment
```

Comment:
Shouldn't this be `ExperimentRapid`?

Reply:
That model won't exist when this pr lands.